src/resources/js/admin-image-field.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable es5/no-arrow-functions */
+/* eslint-disable es5/no-es6-methods */
+jQuery( $ => {
+	let frame;
+
+	$('.tribe-field-image').each(( x, elm ) => {
+		const fieldParent = $(elm),
+			addImgLink    = fieldParent.find( '.tribe-admin__settings-image-field-btn-add' ),
+			removeImgLink = fieldParent.find( '.tribe-admin__settings-image-field-btn-remove' ),
+			imgContainer  = fieldParent.find( '.tribe-admin__settings-image-field-image-container' ),
+			imgIdInput    = fieldParent.find( '.tribe-admin__settings-image-field-input' );
+
+		const setHiddenElements = () => {
+			const imageIsSet = imgIdInput.val() !== '';
+			addImgLink.toggleClass( 'hidden', imageIsSet );
+			removeImgLink.toggleClass( 'hidden', !imageIsSet );
+			imgContainer.toggleClass( 'hidden', !imageIsSet );
+		};
+
+		addImgLink.on('click', e => {
+			e.preventDefault();
+			if ( frame ) {
+				frame.open();
+			}else{
+				// eslint-disable-next-line no-undef
+				frame = wp.media({
+					// eslint-disable-next-line no-undef
+					title: tribe_admin_image_field.select_image_text,
```

Comment:
Like I mentioned, pull this particular set of text variables from `data-` attrs on the main `tribe-field-image`